.github/workflows/_buildpacks-release.yml

```
@@ -0,0 +1,245 @@
+name: _buildpacks-release
+
+on:
+  workflow_call:
+    inputs:
+      app_id:
+        description: Application ID of GitHub application (Linguist)
+        type: string
+        required: true
+      dry_run:
+        required: false
+        type: boolean
+        default: false
+        description: Flag used for testing purposes that prevents actual changes to the heroku/builder repo
+      ip_allowlisted_image:
+        description: The image to use to run jobs that require IP allowlist privileges
+        type: string
+        required: false
+        default: pub-hk-ubuntu-22.04-small
+    secrets:
+      app_private_key:
+        description: Private key of GitHub application (Linguist)
+        required: true
+      cnb_registry_token:
+        required: true
+        description: The token to login to the CNB registry with
```

The CNB registry itself doesn't  have its own auth system, the token is that of a GitHub account:
https://github.com/buildpacks/github-actions#request-add-entry-action

```suggestion
        description: The token of the GitHub user used to interact with the CNB registry
```